projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.holo.craft.modular_single": "单头工具",
+    "tetra.holo.craft.modular_bow": "弓",
+    "tetra.holo.craft.modular_shield": "盾牌",
+    "tetra.holo.craft.slot": "插槽",
+    "tetra.holo.craft.requirements": "需要",
+    "tetra.holo.craft.improvements": "改进",
+    "tetra.holo.craft.stats": "属性",
+    "tetra.holo.craft.variants": "变种",
+    "tetra.holo.craft.materials.stat.primary": "硬度",
+    "tetra.holo.craft.materials.stat.primary.tooltip": "材料的硬度、锋利度，或被塑形成硬质/锋利边缘的能力",
+    "tetra.holo.craft.materials.stat.secondary": "密度",
+    "tetra.holo.craft.materials.stat.secondary.tooltip": "材料的密度、刚性或重量。被挥动时携带的动大小。",
+    "tetra.holo.craft.materials.stat.tertiary": "韧性",
+    "tetra.holo.craft.materials.stat.tertiary.tooltip": "材料的韧性或抗拉强度。材料被弯曲后回弹的速度",
+    "tetra.holo.craft.materials.stat.tool_level": "工具等级",
+    "tetra.holo.craft.materials.stat.tool_level.tooltip": "使用这种材料合成的工具的基础等级",
+    "tetra.holo.craft.materials.stat.tool_efficiency": "工具效率",
+    "tetra.holo.craft.materials.stat.tool_efficiency.tooltip": "使用这种材料合成的工具的基础效率",
+    "tetra.holo.craft.materials.stat.durability": "耐久",
+    "tetra.holo.craft.materials.stat.durability.tooltip": "使用这种材料合成的工具的基础耐久",
+    "tetra.holo.craft.materials.stat.integrity": "完整度",
+    "tetra.holo.craft.materials.stat.integrity.tooltip": "使用这种材料合成的组件的基础完整度增益或消耗（增益 | 消耗）",
+    "tetra.holo.craft.materials.stat.magic_capacity": "魔力",
+    "tetra.holo.craft.materials.stat.magic_capacity.tooltip": "使用这种材料合成的主要组件的魔力容量",
+    "tetra.holo.craft.materials.stat_modifier": "修正",
+    "tetra.holo.craft.materials.stat_modifier.tooltip": "使用这种材料合成的主要组件会提供§e%s§r修正：",
+    "tetra.holo.craft.materials.stat_effect": "效果",
+    "tetra.holo.craft.materials.stat_effect.tooltip": "使用这种材料合成的主要组件会提供§e%s§r效果：",
+    "tetra.workbench.slot_detail.details_tab": "详情",
+    "tetra.workbench.slot_detail.craft_tab": "合成",
+    "tetra.workbench.slot_detail.tweak_tab": "调整",
+    "tetra.workbench.slot_detail.tweak_apply": "应用",
+    "tetra.workbench.module_detail.empty": "空槽位",
+    "tetra.workbench.module_detail.not_tweakable": "调整不可用",
+    "tetra.workbench.schematic_list.empty": "所选插槽没有可用的升级。",
+    "tetra.workbench.schematic_detail.craft": "合成",
+    "tetra.workbench.schematic_detail.back": "返回",
+    "tetra.workbench.schematic_detail.experience": "此操作需要 %d 级经验",
+    "tetra.workbench.schematic_list.temporary": "你的背包中有可用物品",
+    "tetra.repair_action.label": "修理",
+    "tetra.tool.hammer": "锤",
+    "tetra.tool.hammer.requirement": "此操作需要 %d 级的锤",
+    "tetra.tool.axe": "斧",
+    "tetra.tool.axe.requirement": "此操作需要 %d 级的斧",
+    "tetra.tool.pickaxe": "镐",
+    "tetra.tool.pickaxe.requirement": "此操作需要 %d 级的镐",
+    "tetra.tool.shovel": "锹",
+    "tetra.tool.shovel.requirement": "此操作需要 %d 级的锹",
+    "tetra.tool.cut": "切割",
+    "tetra.tool.cut.requirement": "这一步需要一个 %d 级的切割工具。\n§7大部分的剑可以被用于切割。",
+    "tetra.tool.pry": "撬",
+    "tetra.tool.pry.requirement": "这一步需要一个 %d 级的撬具。\n§7羊角头可以被用于撬。",
+    "tetra.tool.available": "你有一 个%d 级的",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "tetra.tool.available": "你有一个 %d 级的",
```